src/main.rs

```diff
@@ -14,6 +14,9 @@ use port_strategy::PortStrategy;
 mod benchmark;
 use benchmark::{Benchmark, NamedTimer};
 
+mod scripts;
+use scripts::{Executable::Nmap, Script};
```

Comment:
Would we have to add every core script here?

Reply:
No this is just for maybe the ones we want to have built in. Like a default nmap scan. The rest of the scripts will be dynamic.. Maybe we don't need anything like that at all.